aws/cloudformation/iam.yml.erb

```diff
@@ -454,6 +439,55 @@ roles.each do |name, config| %>
               - 'arn:aws:s3:::cdo-nces/*'
               - 'arn:aws:s3:::cdo-nces'
 
+  AdvocacyS3AccessPolicy:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      ManagedPolicyName: NewAdvocacyS3AccessPolicy
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Action:
+              - s3:ListAllMyBuckets
+              - s3:GetBucketLocation
+            Resource:
+              - arn:aws:s3:::*
+          - Effect: Allow
+            Action:
+              - s3:ListBucket
+              - s3:ListBucketVersions
+            Resource:
+              - 'arn:aws:s3:::cdo-data-sharing-internal'
+              - 'arn:aws:s3:::cdo-census'
+              - 'arn:aws:s3:::cdo-nces'
+              - 'arn:aws:S3:::cdo-dist'
+          - Effect: Allow
+            Action:
+              - s3:AbortMultipartUpload
+              - s3:DeleteObject
+              - s3:DeleteObjectTagging
+              - s3:DeleteObjectVersion
+              - s3:DeleteObjectVersionTagging
+              - s3:GetObject
+              - s3:GetObjectAcl
+              - s3:GetObjectTagging
+              - s3:GetObjectTorrent
+              - s3:GetObjectVersion
+              - s3:GetObjectVersionAcl
+              - s3:GetObjectVersionForReplication
+              - s3:GetObjectVersionTagging
+              - s3:GetObjectVersionTorrent
+              - s3:PutObject
+              - s3:PutObjectTagging
+              - s3:PutObjectVersionTagging
+              - s3:RestoreObject
+            Resource:
+              - 'arn:aws:s3:::cdo-data-sharing-internal/*'
+              - 'arn:aws:s3:::cdo-census/*'
+              - 'arn:aws:s3:::cdo-nces/*'
+              - 'arn:aws:S3:::cdo-dist/production/assets/advocacy/stateofcs/2023/*'
```

Comment:
maybe these paths should be `t/production/assets/advocacy/stateofcs/*` so that future years can be added without modifying this Policy?

Reply:
Broader question ... `advocacy.code.org` is no longer hosted on Pegasus. Does it still use a platform that can't host PDFs and serve the PDFs with `code.org/assets/advocacy/stateofcs/*` ?

Reply:
Also ... now that we're using `git-lfs` can we just commit these large PDFs to our repository alongside other static content and rely on our assets pipeline to publish them to the `cdo-dist` bucket and fingerprint them?

Reply:
`stateofcs/*` is probably fine, but the ask was specifically for 2023 and 2024, and I wouldn't mind being reminded about this configuration in a year :laughing: 

Correct, Advocacy.code.org is now hosted in LeadPages, which does not support PDF documents, this is the workaround for serving those files until the CMS can take over.

Agreed that we could send these through github. [I mentioned that in the Slack thread.](https://codedotorg.slack.com/archives/C03CK49G9/p1729026901650029?thread_ts=1729008372.047999&cid=C03CK49G9). The entire advocacy path doesn't seem to be part of our asset pipeline, however.